src/commitm.zsh

```diff
@@ -137,7 +145,11 @@ commitm() {
             g) prompt_mod_description="More general";;
         esac
 
-        show_echo "\n$prompt_mod_description prompt: \e[1m\e[36m$(cat "$commit_message_temp_file")\e[0m\n"
+        if [[ -n "$prompt" ]]; then
+            show_echo "\n$prompt prompt: \e[1m\e[36m$(cat "$commit_message_temp_file")\e[0m\n"
+        else
+            show_echo "\n$prompt_mod_description prompt: \e[1m\e[36m$(cat "$commit_message_temp_file")\e[0m\n"
+        fi
```

Comment:
@ellipsis-dev this isn't the right way to set the prompt. you don't need to echo the prompt that the user passed. the prompt that the user passes should just be used for the llm call. please fix